src/bootman/bootman.h

```diff
@@ -350,9 +352,35 @@ static inline void kernel_array_free(void *v)
         nc_array_free(&a, (array_free_func)free_kernel);
 }
 
+/**
+ * Get freestanding initrds (initrds without kernel modules inside)
```

Comment:
A reasonable documentation piece would be:
```
Enumerates freestanding initrds. A "freestanding" initrd is an initrd which is
not associated with a specific version of the kernel and is added to each
boot configuration entry. Such initrd is expected to not contain any kernel
modules. There could be any number of freestanding initrds configured.
They are appended in arbitrary order after the kernel-specific initrd in the
bootloader configuration file.
```

Reply:
doing it